api/PointerEvent.json

```diff
@@ -0,0 +1,212 @@
+{
+  "api": {
+    "PointerEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PointerEvent",
+        "support": {
+          "webview_android": {
+            "version_added": "55"
+          },
+          "chrome": {
+            "version_added": "55"
+          },
+          "chrome_android": {
+            "version_added": "55"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": false
+            },
+            {
+              "version_added": "41",
+              "flag": {
+                "type": "preference",
+                "name": "dom.w3c_pointer_events.enabled",
+                "value_to_set": "true"
+              }
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": false
+            },
+            {
+              "version_added": "41",
+              "flag": {
+                "type": "preference",
+                "name": "dom.w3c_pointer_events.enabled",
+                "value_to_set": "true"
+              }
+            }
+          ],
+          "ie": [
+            {
+              "version_added": "11"
+            },
+            {
+              "version_added": "10",
+              "prefix": "MS",
```

Comment:
Lower case "ms". It is not case-sensitive and it is our convention (We wouldn't use MOZ or WEBKIT, but the lower case variants)

Reply:
I took it from here: https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix
Is that out of date? It specifically instructs uppercase for interfaces.

Reply:
Good point. Forget about my lowercase comments then.